### ISSUE TITLE
RustBridge event in RustFest Barcelona

### ISSUE THREAD

**Issue by jrvidal** [rustbridge/team#32]
https://barcelona.rustfest.eu/

* Coaches: @jrvidal
* Venue: TBD
* Date: TBD (fall 2019)

**Comment by jrvidal** [comment]
cc @badboy

It's still early days, so this is mostly an open call for other people interested in coaching or helping out.

I do have a question: is it expected that attendants to a RustBridge-within-RustFest _must_ be RustFest attendants, i.e. people with a RustFest ticket? Or can folks enroll for this particular workshop only?

**Comment by komaeda** [comment]
@jrvidal Nope! RustBridge attendees can have a conference ticket, but don't have to.

**Comment by Dylan-DPC-zz** [comment]
@jrvidal I can help out and open to coaching if needed.

**Comment by jrvidal** [comment]
Updated with dates and venue! Main detail to keep in mind: workshops will be spread among two days.

**Comment by komaeda** [comment]
I'm still figuring out if I'll go, but if I do, I'll be open to helping out as well. @jrvidal congrats though!

**Comment by badboy** [comment]
Hey! Sorry for not reaching out earlier.
Yes, indeed we spread out the workshops across the 2 days.
I would assume that the rustbridge workshop would just continue on the second day (given that for rustbridge the full time was always useful)

I hope this works for you. I'm always open to speak more closely and we can coordinate further communication.

**Comment by jrvidal** [comment]
@Dylan-DPC @komaeda How are your plans for RustFest? :smile:

**Comment by jrvidal** [comment]
@rahul-thakoor I think you expressed an interest in helping out? :slightly_smiling_face:

**Comment by Dylan-DPC-zz** [comment]
@jrvidal nothing planned yet. Was waiting for some things to resolve so that I can get the trip covered by the company.

also maybe we should catch up somewhere (preferably on discord ) to discuss further steps.

**Comment by jrvidal** [comment]
Yep, Discord works, I'm `_rvidal` in the official Rust one. Also there's the RustBcn slack, which might also be an appropriate venue (and lets us create rooms). 

cc @fmoko (hope you're the right person, I followed several github user redirects)

**Comment by manyinsects** [comment]
@jrvidal I'm definitely gonna be there, helping out (this is komaeda's new account btw)

**Comment by Dylan-DPC-zz** [comment]
Uhmm rustfest tickets sold out :/ now need to find some "backdoor" ;)

**Comment by badboy** [comment]
Send me an email to janerik@rustfest.eu with who is planned to be coaches for the RustBridge. We will be able to get you on there ;)

**Comment by kinnison** [comment]
I was directed here by @Dylan-DPC to volunteer if you still want more coaches.  I'll be in Barcelona anyway, despite lacking a ticket to Rustfest, so if @badboy still has flexibility and I'm wanted, I'd appreciate being put on whatever list will let me help.  It'll keep me out of bars at least :D
